concepts/Common/Licensing/25 How to Use the License Key.md

```diff
@@ -0,0 +1,7 @@
+The license key must be specified in `GlobalConfig`. You can use the same license key for all your applications.
```

Comment:
It seems that we call this just `config`: https://js.devexpress.com/React/Documentation/ApiReference/Common/Utils/#configconfig

I would add this link here as well